src/App.tsx

```diff
@@ -123,15 +176,52 @@ function App () {
         void getMapsAPIKeyFromCRM()
     }, [])
 
+    function toggleMapSize (mapFullScreen: boolean) {
```

Comment:
can we go with `isMapFullScreen`? Prefer is/has prefix for boolean vars: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/boolean-prop-naming.md